### PR TITLE
Apply checkstyle for java lint

### PR DESCRIPTION
**Pull request by KeonHee**
- [x] Add checkstyle config
- [x] Apply checkstyle to code 

`./gradlew build` contains the checkstyle command.

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -46,3 +47,15 @@ publishPlugin {
...
+
+checkstyle {
+    configFile = file("${rootDir}/config/checkstyle/checkstyle.xml")
+    toolVersion = 8.35
+}
+
+tasks.withType(Checkstyle) {
+    reports {
+        xml.enabled false
+        html.enabled true
```
Comment (style95) [inline build.gradle:59]: How can we access this report?

Comment (KeonHee) [reply build.gradle:59]: When `./gradlew build` is running, The checkstyle is checking and showing as below. There is no need to access the report actually.

image

Comment (KeonHee) [reply build.gradle:59]: And you can show the cause of failure in github action
 - https://github.com/naver/openwhisk-intellij-plugin/runs/1019621549?check_suite_focus=true
image

Comment (style95) [reply build.gradle:59]: ok I just asked because it is enabled.

**Review by style95 — APPROVED** [review]

**Comment by KeonHee** [comment]
Merged. 

Thank for review @style95

**Pull request merged**